modules/collator-selection/src/lib.rs

```diff
@@ -445,9 +449,12 @@ pub mod pallet {
 				.checked_sub(&T::Currency::minimum_balance())
 				.unwrap_or_default()
 				.div(2u32.into());
+
 			// `reward` is half of pot account minus ED, this should never fail.
```

Comment:
move this comment the line above the transfer otherwise it looks like the `>=` should never fail